fe/src/components/campaign/content.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="flex flex-col gap-6 sm:w-full lg:w-full">
```

Comment:
so it can be w-full regardless of screen right?
```suggestion
  <div class="flex flex-col gap-6 w-full">
```

Reply:
yes sir

> By default, styles applied by rules like `md:flex` will apply at that breakpoint and stay applied at larger breakpoints.

https://tailwindcss.com/docs/responsive-design#targeting-a-breakpoint-range